src/SB/Game/zNPCTypeKingJelly.cpp

```diff
@@ -1314,6 +1319,38 @@ void zNPCKingJelly::BUpdate(xVec3* pos)
     zNPCCommon::BUpdate(pos);
 }
 
+S32 zNPCKingJelly::SysEvent(xBase* from, xBase* to, U32 toEvent, const F32* toParam,
+                            xBase* toParamWidget, S32* handled)
+{
+    U32 uVar1;
```

Comment:
A better name can be inferred.
Example:
```suggestion
    U32 ret;
```